File: backend/local-api-module/src/main/java/org/vyatsu/localApiModule/dto/response/api/ToDoTaskDto.java
package org.vyatsu.localApiModule.dto.response.api;

import lombok.Value;

import java.io.Serializable;
import java.time.LocalDateTime;

/**
 * DTO for {@link org.vyatsu.localApiModule.entity.user.ToDoTask}
 */
@Value
public class ToDoTaskDto implements Serializable {
    Long id;
    String title;
    String description;
    LocalDateTime dueDate;
    Boolean isCompleted;
    LocalDateTime createdAt;
}

File: app/src/main/java/com/bytebusters/android/mpsystem/dto/response/api/ToDoTaskDto.kt
package com.bytebusters.android.mpsystem.dto.response.api

import java.time.LocalDateTime

class ToDoTaskDto (
    var id: Long? = null,
    var title: String? = null,
    var description: String? = null,
    var dueDate: LocalDateTime? = null,
    var isCompleted: Boolean? = null,
    var createdAt: LocalDateTime? = null
)